ymer-test-junit5/src/main/java/com/avanza/ymer/test/YmerConverterTestBase.java

```diff
@@ -138,6 +147,34 @@ private MongoConverter createMongoConverter() {
 		return converter;
 	}
 
+	private static void ensureSpaceId(Object spaceObject) {
```

Comment:
would it be possible to re-use the method from juni4 version instead of duplicating the code?

Reply:
I don't want the JUnit 5 module to depend on the JUnit 4 module, but I've created a new shared test module for them to share code.
I moved a couple of duplicate classes there too.